### PR TITLE
Fix/docs

### PR DESCRIPTION
**Pull request by DmitryRomaniuk**

### CODE REVIEW

File: server/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "target": "es2017",
     "outDir": "./dist",
+    "baseUrl": "./",
```
Comment (apalchys) [inline server/tsconfig.json:6]: unfortunately, our current setup is not compatible with this change. If you create a production Docker image it won't work.

File: server/CONTRIBUTING.md [hunk 2]
```diff
@@ -19,7 +19,8 @@
 #### Prerequisites
```
Comment (apalchys) [inline server/CONTRIBUTING.md:19]: Please add a link to `server/CONTRIBUTING.md` in `README`

File: server/tsconfig.json (hunk 1 above)
Comment (DmitryRomaniuk) [reply server/tsconfig.json:6]: thx fixed

File: server/CONTRIBUTING.md (hunk 2 above)
Comment (DmitryRomaniuk) [reply server/CONTRIBUTING.md:19]: thx fixed

File: README.md [hunk 3]
```diff
@@ -32,11 +32,10 @@ The [web application](https://app.rs.school) which powers [Rolling Scopes School
...
   - Run `npm install` (installs dependencies in the root folder and `client` / `server` folders.)
-  - Run `npm run db:up` (starts local database)
-  - Run `npm run db:restore`  (restore a test DB snapshot)
-  - Make a copy of `server/.env.example` and rename it to `server/.env`
-  - Run `npm start` (starts application by running Next.js and REST API server)
-  - Open `https://localhost:3000` in a browser
+  - Read documentation about server `server/CONTRIBUTING.md` at least from 1 to 7 paragraphs
+  - Open terminal, navigate into the directory `client` and run `npm start` (starts application by running Next.js and REST API server)
+  - Open `http://localhost:3000` in a browser
+  - If you want to look routes on the server, you can open `http://localhost:3001/swagger` in a browser
```
Comment (apalchys) [inline README.md:38]: Please leave as it was before. It was working guide how to set up local env.
Please don't mention swagger route - it's obsolete and confusing. 

I just recommend you to add additional point with reference to `server/CONTRIBUTING.md`

e.g.
```
- Make a copy of `server/.env.example` and rename it to `server/.env`
- Run `npm start` (starts application by running Next.js and REST API server)
- Open `https://localhost:3000` in a browser
- If you faced with any issue to set up server, please read more details guide here: server/CONTRIBUTING.md
```

File: server/CONTRIBUTING.md [hunk 4]
```diff
@@ -35,29 +36,30 @@ $ git clone git@github.com:[username]/rsschool-api.git
...
     ``` command-line
-    
         git clone git@github.com:rolling-scopes/rsschool-scripts.git
         cd rsschool-scripts/development
         docker-compose up
     ```
     For more information about Docker Compose, please check [Docker Compose documentation](https://docs.docker.com/compose/)
-    
+
+    * If you have some problems with connection to db, please check env variables password, user and db in [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts/blob/master/development/docker-compose.yml)
```
Comment (apalchys) [inline server/CONTRIBUTING.md:62]: please reference to this file: `rsschool-app/setup/docker-compose.yml`
`rsschool-scripts` repo is not used anymore

File: server/CONTRIBUTING.md [hunk 5]
```diff
@@ -35,29 +36,30 @@ $ git clone git@github.com:[username]/rsschool-api.git
...
+        RSSHCOOL_API_PG_HOST
+        RSSHCOOL_API_PG_USERNAME
+        RSSHCOOL_API_PG_PASSWORD
+        RSSHCOOL_API_PG_DATABASE
     ```
 
-    * You may run MongoDB locally with pre-imported test data using [Docker Compose file](https://github.com/rolling-scopes/rsschool-scripts/blob/master/development/docker-compose.yml) from [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts) repository.
-    Following commands will run MongoDB locally binded to 27107 port and will import test `Users` and `Courses` data.
+    * You should run Postgres locally with pre-imported test data using [Docker Compose file](https://github.com/rolling-scopes/rsschool-scripts/blob/master/development/docker-compose.yml) from [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts) repository.
+    Following commands will run Postgres locally binded to 5432 port and will import test `Users` and `Courses` data.
```
Comment (apalchys) [inline server/CONTRIBUTING.md:54]: - `npm run db:up` runs database locally using docker
- `npm run db:restore` restore test copy of DB locally.

`rsschool-scripts` repo is not used anymore

File: server/CONTRIBUTING.md [hunk 6]
```diff
@@ -19,7 +19,8 @@
 #### Prerequisites
 - NodeJS LTS
 - Python 2.7+
-- MongoDB 3.6+
+- [Docker](https://docs.docker.com/engine/install/) + [Docker-compose](https://docs.docker.com/compose/install/)
+- [rsschool-scripts](https://github.com/rolling-scopes/rsschool-scripts)
```
Comment (apalchys) [inline server/CONTRIBUTING.md:23]: `rsschool-scripts` repo is not used anymore

File: server/CONTRIBUTING.md [hunk 7]
```diff
@@ -19,7 +19,8 @@
 #### Prerequisites
 - NodeJS LTS
 - Python 2.7+
```
Comment (apalchys) [inline server/CONTRIBUTING.md:21]: please remove Python 2.7+. It's not mandatory anymore.

**Pull request merged**